SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -230,6 +232,23 @@ void USpatialReceiver::HandleActorAuthority(Worker_AuthorityChangeOp& Op)
 		return;
 	}
 
+	if (Op.component_id == SpatialConstants::CLIENT_RPC_ENDPOINT_COMPONENT_ID
+		&& Op.authority == WORKER_AUTHORITY_AUTHORITATIVE)
+	{
+		if (!NetDriver->IsServer())
```

Comment:
I believe this is not necessary. The server can never have authority over this component.

Reply:
Maybe you could make this an `ensure`?

Reply:
Changed to ensure.